qemu/tests/qemu_guest_agent.py

```diff
@@ -3477,6 +3506,8 @@ def get_blocks():
 
         session = self._get_session(params, None)
         self._open_session_list.append(session)
+        if params.get("os_type") == "linux":
+            self.gagent_setsebool_value('on', params, self.vm)
```

Comment:
This one does not need set it back?

Reply:
this is another reminder, I should move this to linux block,this is windows module, and I reset it at fstrim function (linux),thanks remind me that.

Reply:
@vivianQizhu and this one.